src/context/GrowerContext.js

```diff
@@ -0,0 +1,125 @@
+import React, { useState, createContext } from 'react';
```

Comment:
git has picked up this renaming as a deletion and creation of a new file.
Are you able to redo this file using a `git mv` command so we keep the history and can compare?